### DISCUSSION TITLE
Syscall & UserSpace

### DISCUSSION THREAD

**Discussion by Yo-yo-ooo** [marceldobehere/MaslOS#15] [Q&A] [answered]
How do you plan to make system calls and user space

**Comment by marceldobehere** [comment, marked as answer]
I do not plan on adding syscalls and it will all be kernel space for now. Though MAAL apps have their own kind of memory region. I plan on improving MAAL and adding more OS APIs so that all apps will be in MAAL and so userspace shouldn't be a problem.
